dts/common/nordic/nrf9280.dtsi

```diff
@@ -0,0 +1,1207 @@
+				clocks = <&canpll>;
+				bosch,mram-cfg = <0x0 28 8 3 3 0 1 1>;
+				status = "disabled";
+			};
+
+			/* TODO check */
+			can121: can@8db000 {
+				compatible = "nordic,nrf-can";
+				reg = <0x8db000 0x400>, <0x2fbef800 0x800>, <0x2fbe8000 0x7800>;
+				reg-names = "wrapper", "m_can", "message_ram";
+				interrupts = <216 NRF_DEFAULT_IRQ_PRIORITY>;
+				clocks = <&canpll>;
+				bosch,mram-cfg = <0x0 28 8 3 3 0 1 1>;
+				status = "disabled";
+			};
+
+			dppic120: dppic@8e1000 {
+				compatible = "nordic,nrf-dppic-global";
+				reg = <0x8e1000 0x1000>;
+				status = "disabled";
+			};
+
+			timer120: timer@8e2000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x8e2000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <226 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				max-frequency = <DT_FREQ_M(320)>;
+				prescaler = <0>;
+			};
+
+			timer121: timer@8e3000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x8e3000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <227 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				max-frequency = <DT_FREQ_M(320)>;
+				prescaler = <0>;
+			};
+
+			pwm120: pwm@8e4000 {
+				compatible = "nordic,nrf-pwm";
+				reg = <0x8e4000 0x1000>;
+				status = "disabled";
+				interrupts = <228 NRF_DEFAULT_IRQ_PRIORITY>;
+				#pwm-cells = <3>;
+			};
+
+			spi120: spi@8e6000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x8e6000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <230 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(32)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				rx-delay-supported;
+				rx-delay = <1>;
+			};
+
+			uart120: uart@8e6000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x8e6000 0x1000>;
+				status = "disabled";
+				interrupts = <229 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			spi121: spi@8e7000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x8e7000 0x1000>;
+				status = "disabled";
+				easydma-maxcnt-bits = <15>;
+				interrupts = <231 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-frequency = <DT_FREQ_M(32)>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				rx-delay-supported;
+				rx-delay = <1>;
+			};
+
+			cpuppr_vpr: vpr@908000 {
+				compatible = "nordic,nrf-vpr-coprocessor";
+				reg = <0x908000 0x1000>;
+				status = "disabled";
+				cpu = <13>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+				ranges = <0x0 0x908000 0x4000>;
+
+				cpuppr_vevif_tx: mailbox@0 {
+					compatible = "nordic,nrf-vevif-task-tx";
+					reg = <0x0 0x1000>;
+					status = "disabled";
+					#mbox-cells = <1>;
+					nordic,tasks = <16>;
+					nordic,tasks-mask = <0xfffffff0>;
+				};
+
+				cpuppr_clic: interrupt-controller@1000 {
+					compatible = "nordic,nrf-clic";
+					reg = <0x1000 0x3000>;
+					status = "disabled";
+					#interrupt-cells = <2>;
+					interrupt-controller;
+					#address-cells = <1>;
+				};
```

Comment:
> I don't think 0x909000 is the correct CLIC address. Anyway, this peripheral is internal VPR peripheral so probably should be removed from this common file.

nrf-regtool depends on CLIC nodes being visible from any core. We use that for interrupt mapping between cores.

Also, judging by the address of this node on nRF9230 EngB (0x5F909000 is correct), it appears to be accessible to PPR's owner, so I think it can stay here for now.

(Note: on other chips, a CLIC may be private to a given VPR and inaccessible to all other cores. This can still be expressed using the `ranges` property - or lack thereof - on the parent bus, but the CLIC node itself would have to be moved.)

Reply:
> Also, judging by the address of this node on nRF9230 EngB (0x5F909000 is correct), it appears to be in the global domain and accessible to PPR's owner, so I think it can stay here for now.

Can you point me to the IPS chapter describing this address? I could not find it. That's why I suggested removing it.

Reply:
Thanks for clarifying. The address is correct and my comment is irrelevant.